apps/client/src/features/rundown/rundown-header/RundownHeader.tsx

```diff
@@ -11,13 +13,31 @@ import RundownMenu from './RundownMenu';
 
 import style from './RundownHeader.module.scss';
 
-export default function RundownHeader() {
+interface RundownProps {
+  data: RundownCached;
+}
+
+export default function RundownHeader({ data }: RundownProps) {
+  const { rundown } = data;
   const appMode = useAppMode((state) => state.mode);
+  const clipBoard = useAppMode((state) => state.eventClipBoard);
```

Comment:
Suggestion: should we name this something else? I spent some time looking for your clipboard implementation... it is good to have clear expectations of what is happening. How about just copiedEvent?

Also, should this not be together with the event swap feature? Should we make a new store for this?

Reply:
Yeah sounds good